Matcher/ContentBased/Pjax.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This file is part of the eZ PlatformUI package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Matcher\ContentBased;
```

Comment:
I would expect the matcher to be in `PlatformUIBundle\Matcher`, as it is not Content Based. Also, I'd name it `PjaxRequest`, as it matches a, well, pjax requests :)